Hearthstone Deck Tracker/Windows/OverlayWindow.MouseOverDetection.cs

```diff
@@ -85,15 +87,35 @@ private void UpdateMouseOverDetectionRegions(List<Entity> oppBoard, List<Entity>
 					AddCardDebugOverlay(_playerHand[i], GetPlayerCardPosition(i, playerHandCount));
 			}
 
-			var leaderboardTop = Height * 0.15;
 			for(var i = 0; i < _leaderboardIcons.Count; i++)
 			{
 				_leaderboardIcons[i].Visibility = isGameOver || !_game.IsBattlegroundsMatch
 					? Visibility.Collapsed
 					: Visibility.Visible;
-				Canvas.SetTop(_leaderboardIcons[i], leaderboardTop + BattlegroundsTileHeight * i);
+				Canvas.SetTop(_leaderboardIcons[i], LeaderboardTop + BattlegroundsTileHeight * i);
 				Canvas.SetLeft(_leaderboardIcons[i], Helper.GetScaledXPos(0.001 * (_leaderboardIcons.Count - i - 1), (int)Width, ScreenRatio));
 			}
+
+			PositionDeadForText();
+		}
+
+		internal void PositionDeadForText()
+		{
+			var leftAdjust = .00075;
+			
+			for(int i = 0; i < _leaderboardDeadForText.Count; i++)
+			{
+				Canvas.SetTop(_leaderboardDeadForText[i], LeaderboardTop + BattlegroundsTileHeight * i);
+				Canvas.SetLeft(_leaderboardDeadForText[i], Helper.GetScaledXPos(leftAdjust * (_leaderboardDeadForText.Count - i - 1), (int)Width, ScreenRatio));
+				Canvas.SetTop(_leaderboardDeadForTurnText[i], LeaderboardTop + BattlegroundsTileHeight * i);
+				Canvas.SetLeft(_leaderboardDeadForTurnText[i], Helper.GetScaledXPos(leftAdjust * (_leaderboardDeadForTurnText.Count - i - 1), (int)Width, ScreenRatio));
+			}
+
+			if(nextOpponentLeaderboardPosition != null)
+			{
+				Canvas.SetLeft(_leaderboardDeadForText[(nextOpponentLeaderboardPosition ?? 0)-1], Helper.GetScaledXPos(leftAdjust * (_leaderboardDeadForText.Count - (nextOpponentLeaderboardPosition ?? 0) - 1 - 1) + .025, (int)Width, ScreenRatio));
```

Comment:
You should and can get rid of the null coalescing operator here. For one because you check for null above and also since if it was null you would throw an `IndexOutOfRangeException` with `_leaderboardDeadForText[-1]`